art/config.py

```diff
@@ -0,0 +1,45 @@
+import json
+import logging
+import os
+
+import numpy as np
+
+logger = logging.getLogger(__name__)
+
+
+# -------------------------------------------------------------------------------------------- DEFUALT PACKAGE CONFIGS
+
+
+ART_NUMPY_DTYPE = np.float32
+
+_folder = os.path.expanduser('~')
+if not os.access(_folder, os.W_OK):
+    _folder = '/tmp'
+_folder = os.path.join(_folder, '.art')
+
+_config_path = os.path.expanduser(os.path.join(_folder, 'config.json'))
+if os.path.exists(_config_path):
+    try:
+        with open(_config_path) as f:
+            _config = json.load(f)
+    except ValueError:
+        _config = {}
+
```

Comment:
Would it be possible to check here if `_config` contains a key `DATA_PATH` and if yes remove it and add the key `ART_DATA_PATH` with the value received for `DATA_PATH`  and overwrite config.json?